PWGHF/TableProducer/HFTrackIndexSkimsCreator.cxx

```diff
@@ -27,8 +27,15 @@
 #include "Common/DataModel/TrackSelectionTables.h"
 #include "ReconstructionDataFormats/V0.h"
 #include "PWGHF/Utils/UtilsDebugLcK0Sp.h"
+#include "DetectorsVertexing/PVertexer.h"     // for PV refit
+#include "ReconstructionDataFormats/Vertex.h" // for PV refit
+#include "CCDB/BasicCCDBManager.h"            // for PV refit
+#include "DataFormatsParameters/GRPObject.h"  // for PV refit
+#include "DetectorsBase/Propagator.h"         // for PV refit
+#include "DetectorsBase/GeometryManager.h"    // for PV refit
 
 #include <algorithm>
+#include <iostream>
```

Comment:
@vkucera looking back at the code probably it is not needed, indeed. This shall be a refuse because at the beginning I was going to use `std::cout` for the debug instead of `LOG`. In the end I left `LOG`, this was there even in other parts of the code I did not implemented by myself.